services/metadactyl-clj/src/metadactyl/core.clj

```diff
@@ -63,15 +63,16 @@
 
 (defn -main
   [& args]
-  (require 'metadactyl.routes.api)
-  (let [app (eval 'metadactyl.routes.api/app)
-        {:keys [options arguments errors summary]} (ccli/handle-args config/svc-info args cli-options)]
-    (when-not (fs/exists? (:config options))
-      (ccli/exit 1 (str "The config file does not exist.")))
-    (when-not (fs/readable? (:config options))
-      (ccli/exit 1 "The config file is not readable."))
-    (load-config-from-file (:config options))
-    (tasks/set-logging-context! config/svc-info)
-    (tasks/schedule-tasks)
-    (log/warn "Listening on" (config/listen-port))
-    (jetty/run-jetty app {:port (config/listen-port)})))
+  (tc/with-logging-context config/svc-info
+    (require 'metadactyl.routes.api)
+    (let [app (eval 'metadactyl.routes.api/app)
+          {:keys [options arguments errors summary]} (ccli/handle-args config/svc-info args cli-options)]
+      (when-not (fs/exists? (:config options))
+        (ccli/exit 1 (str "The config file does not exist.")))
+      (when-not (fs/readable? (:config options))
+        (ccli/exit 1 "The config file is not readable."))
+      (load-config-from-file (:config options))
+      (tasks/set-logging-context! config/svc-info)
```

Comment:
Ah! I am not familiar with refs, but it does look like it's duplication.
Once we are done with this first pass of reviews, I'll be splitting up the work on a service-by-service basis. I can look into removing this when I cycle around to metadactyl.